index.html

```diff
@@ -6,7 +6,325 @@
   </head>
   <body>
 
-    <p>Репозиторий создан для обучения на профессиональном онлайн‑курсе «<a href="https://htmlacademy.ru/intensive/htmlcss">HTML и CSS. Профессиональная вёрстка сайтов</a>».</p>
+    <header class="page-header">
+      <h1 class="visually-hidden">Интернет-магазин Девайс</h1> 
+      <a href="index.html"><img src="#" class="header-logo" alt="логотип Девайс"></a>
+        
+      <nav class="header-navigation">
+        <ul class="navigation-list navigation-user">
+          <li class="navigation-item">
+            <a class="navigation-link" href="#">Поиск по сайту</a>
+          </li>
+          <li class="navigation-item">
+            <a class="navigation-link" href="#">Вoйти</a>
+          </li>
+          <li class="navigation-item">
+            <a class="navigation-link" href="#">Сравнить</a>
+          </li>
+          <li class="navigation-item">
+            <a class="navigation-link" href="#">Корзина</a>
+          </li>
+        </ul>
+
+        <ul class="navigation-list">
+          <li class="navigation-item">
+            <a href="catalog.html">Каталог товаров</a>
```

Comment:
Форматирование: `button` находится на одном уровне вложенности с `a`
```suggestion
 <a href="catalog.html">Каталог товаров</a>
 <button class="dropbtn"></button>
```